brig/cmd/brig/commands/project_create.go

```diff
@@ -0,0 +1,537 @@
+package commands
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/AlecAivazis/survey"
+	"github.com/Masterminds/goutils"
+	"github.com/spf13/cobra"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+
+	"github.com/Azure/brigade/pkg/brigade"
+	"github.com/Azure/brigade/pkg/storage/kube"
+)
+
+const projectCreateUsage = `Create a new project.
+
+Create a new project by answering questions or supplying a configuration file.
+
+Project names are typically in the form 'ORG/PROJECT' or 'USER/PROJECT". For
+example, Azure/brigade or deis/empty-testbed.
+
+A Brigade project provides a context in which a brigade.js file is executed.
+Projects are frequently tied to (Git) source code repositories, and contain
+configuration data such as secrets, Kubernetes-specific directives, and
+settings for gateways or workers.
+
+If a file is provided (-f), then the project defaults will be set from the file.
+
+The user will be prompted to answer questions to build the configuration. If -f
+is specified in conjunction with -x/--no-prompts, then the values in the file will
+be used without prompting the user for any changes.
+
+`
+
+var (
+	projectCreateConfig      = ""
```

Comment:
dont'e need to set these to their default values since `.StringVarP` and `.BoolVarP` already do that after `.ParseFlags()` is called. Should be just `projectCreateConfig string`